src/Server.cpp

```diff
@@ -51,10 +55,22 @@ Server::~Server()
 		m_epollWrapper.removeEvent(iter->first);
 		close(iter->first);
 	}
+
+	for (std::map<int, Connection*>::iterator iter = m_cgiConnections.begin(); iter != m_cgiConnections.end(); ++iter) {
+		m_epollWrapper.removeEvent(iter->first);
+		close(iter->first);
+	}
 }
 
 /* ====== GETTERS ====== */
 
+/**
+ * @brief Getter for epollWrapper.
+ *
+ * @return EpollWrapper& epollwrapper object.
+ */
+EpollWrapper& Server::getEpollWrapper() { return m_epollWrapper; }
+
```

Comment:
Would fit better under DISPATCH TO EPOLLWRAPPER and have it call `getEpollFd()` directly

Reply:
Done